packages/2019-template/src/components/DemoCard/index.js

```diff
@@ -33,6 +33,7 @@ class DemoCard extends Component {
 }
 
 DemoCard.displayName = "DemoCard";
+DemoCard.tags = demoCardMeta().tags;
```

Comment:
There's got to be a better way to pass these tags to the card registry without adding this line every time. @BJHunnicutt any ideas?

Reply:
@Dianna if you have any ideas on this, would be interested in your thoughts

Reply:
Actually, this may be fine. Having this be explicit lets us comment this out until we're ready to expose cards through the tag system